### ISSUE TITLE
Docker container in Dockerhub

### ISSUE THREAD

**Issue by glatard** [rhr-pruim/ICA-AROMA#5]
It would be very useful if you could create an automated build of the Docker container on Dockerhub (https://hub.docker.com). It takes only a few clicks and it's free. Other people cannot do it because it requires permissions to your Github account.

**Comment by maartenmennes** [comment]
Given that the official version is in Raimon's github account, how would I go about creating this? In the docker automated build I can only choose my own repositories.

**Comment by glatard** [comment]
I guess Raimon has to do that then.

**Comment by jlhanson5** [comment]
A slightly related question, there looks to be a Dockerfile in the current repo... but I don't see guidance on the specific syntax? Is it just the same basic command line inputs? 

I kept trying:
`python2.7 ~/Volumes/Hanson/Training_Resources/ICA-AROMA/ica-aroma-via-docker.py -in ~/Volumes/Hanson/Duke_PAC/proc/140313_18263/fmri/rest/ICA_AROMA/20140313_18263_REST_LAS_st_mcfr_brain.nii.gz -out ~/Volumes/Hanson/Duke_PAC/proc/140313_18263/fmri/rest/ICA_AROMA/ -a ~/Volumes/Hanson/Duke_PAC/proc/140313_18263/fmri/rest/ICA_AROMA/func2struct.mat -w ~/Volumes/Hanson/Duke_PAC/proc/140313_18263/fmri/rest/ICA_AROMA/my_nonlinear_transf.nii.gz -mc ~/Volumes/Hanson/Duke_PAC/proc/140313_18263/fmri/rest/20140313_18263_REST_LAS_st_mcfr_brain_norm.mat`

but ran into different various issues. Any thoughts are greatly appreciated!

**Comment by maartenmennes** [comment]
I'm currently still a noob when it comes to docker. This function was added by @chrisfilo - perhaps he can chime in?

**Comment by jlhanson5** [comment]
Yes, I am a super noob w/ Docker too... so any assistance is greatly appreciated. I can pull the docker image, but can't figure out the specific arguments that are needed or what the syntax is, etc. Thanks much!

**Comment by chrisfilo** [comment]
I did not put this container on Docker Hub so I am not sure what are you pulling. You would need to specify which container are you pulling (or building), what command you are trying to run, and what errors (full trace) you are getting for people to be able to help you.

**Comment by jlhanson5** [comment]
That might explain it, I was trying to use ica-aroma-via-docker.py. I also tried to use this docker container:
https://hub.docker.com/r/mcin/ica-aroma/
